spec/bolt/module_installer/puppetfile_spec.rb

```diff
@@ -112,5 +112,21 @@
         )
       end
     end
+
+    context 'with missing module version' do
+      let(:spec)    { double('spec', satisfied_by?: true) }
+      let(:version) { nil }
+      let(:mod)       {
+        [double('mod', name: name, version: version,
+                       is_a?: true, to_spec: "mod '#{name}'")]
+      }
```

Comment:
```suggestion

      let(:mod) do
        [double('mod', name: name, version: version, is_a?: true, to_spec: "mod '#{name}'")]
      end
```

Since this is too long for one line wrap it in `do .. end`.